src/containers/Detail/Detail.scss

```diff
@@ -0,0 +1,42 @@
+.detail {
+  display: flex;
+  margin: auto;
+  transform: translateY(50%);
+  width: 1000px;
+  padding: 16px;
+  box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;
+
+  .detail__item {
+    flex-grow: 1;
+    display: flex;
+    align-items: center;
+    height: 260px;
+    margin: 0 8px;
+    &.detail__logo {
+      .image {
+        height: 240px;
+        width: 240px;
+      }
+    }
+    &.detail__info {
+      .detail__info__name {
```

Comment:
We are using BEM. It's block__element--modifier.
We shouldn't use block__element__element__element...

Reply:
i will change from .detail__into__name to .into__name and nested into .detail__into